### DISCUSSION TITLE
Feature request log specific commands

### DISCUSSION THREAD

**Discussion by Misio12320** [EssentialsX/Essentials#4109] [Ideas and Feature Suggestions] [answered]
Add a feature into config to log commands. I mean log to send to console "user executed %command%", default enabled and can be disabled in "disabled log command to console true:
- gmc
If player execute command gmc it should not send message into console" player executed command gmc"

**Comment by JRoy** [comment, marked as answer]
This is not really something Essentials itself will add, look for a 3rd party plugin.
